src/portal/src/lib/components/cron-schedule/cron-schedule.component.scss

```diff
@@ -20,18 +16,14 @@
       }
     }
     button {
-      margin: 35px 20px 0 200px;
+      margin: 35px 20px 0;
     }
   }
   
   .setting-wrapper {
     position: relative;
-    > span:first-child {
-      width: 200px;
-    }
-
     .confirm-button {
-      margin: 20px 0px 0 200px;
+      margin: 20px 0 0;
```

Comment:
Why just specify 3 parameter instead of 4?

Reply:
here is the fourth parameter :  @Input() buttonMarginLeft: string = '200px';

Reply:
default value  is 200px  ,  and it can be customized